refactor(core): a couple of minor changes to the early-event-contract

alan-agius4 opened the pull request:
This changes include
- Using multi line comments to write JSDoc comments which improves DX
- Use `this.container` instead of `window.document.documentElement`

AndrewKushnir commented on packages/core/primitives/event-dispatch/src/earlyeventcontract.ts:
```diff
@@ -52,7 +52,7 @@ export class EarlyEventContract {
           event,
           event.target as Element,
-          window.document.documentElement,
+          this.container,
```
nit: can we refer to the `container` without `this` (to save some bytes)?
```suggestion
          container,
```

AndrewKushnir reviewed (approved):
👍

alan-agius4 commented:
[TGP](http://test/OCL:631401009:BASE:631662186:1715147785013:355b6dde)

alan-agius4 committed:
refactor(core): a couple of minor changes to the early-event-contract

This changes include
- Using multi line comments to write JSDoc comments which improves DX
- Use `this.container` instead of `window.document.documentElement`

alan-agius4 committed:
fixup! refactor(core): a couple of minor changes to the early-event-contract

alan-agius4 commented:
[TGP](http://test/OCL:631401009:BASE:631643683:1715180819562:3f332651)

atscott commented:
This PR was merged into the repository by commit c0012865eba3717dc9d1d9d3a9d5b7df909897ff.
